src/hdf5/utils.cpp

```diff
@@ -0,0 +1,32 @@
+//
+// ICRAR - International Centre for Radio Astronomy Research
+// (c) UWA - The University of Western Australia, 2017
+// Copyright by UWA (in the framework of the ICRAR)
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+//
+
+/**
+ * @file
+ *
+ * Various utilities for interacting with the HDF5 API
+ */
+
+namespace shark {
+namespace hdf5 {
+
+// Nothing... yet
```

Comment:
Haha no, I'll remove it. I kept going backwards and forwards having something in here so I was leaving it to avoid all the other setup if I added another implementation back again :laughing: